packages/utils/src/modules-sdk/internal-module-service-factory.ts

```diff
@@ -63,6 +63,48 @@ export function internalModuleServiceFactory<
       return keys.map((k) => data[k]).join("_")
     }
 
+    static applyDefaultOrdering(config: FindConfig<any>) {
+      if (config.order) {
+        return
+      }
+
+      const order: object[] = []
+
+      const primaryKeys = AbstractService_.retrievePrimaryKeys(model)
+      const mainEntityOrder = {}
+      primaryKeys.forEach((pk) => {
+        mainEntityOrder[pk] = "ASC"
+      })
+      order.push(mainEntityOrder)
+
+      if (config.relations?.length) {
+        config.relations.forEach((relation) => {
+          const relation_ = (
```

Comment:
Rather than using "private" models, an easier to read approach would be to be more concrete about the variables. For example, the `relation` in the array would be a `relationKey` or `relationName`, and not a relation object. Then you can just name this `relation` without any underscores.